seed.js

```diff
@@ -0,0 +1,140 @@
+//add user helper function
+const addUsers = (i, namesList, isAdmin) => {
+  let [firstName, lastName] = namesList[i].split(' ');
+  let email = `${firstName.toLowerCase()}.${lastName.toLowerCase()}@gmail.com`;
+  let password = '123'
+  let id = users.length + 1;
+  users.push({
+    id,
+    firstName,
+    lastName,
+    email,
+    password,
+    isAdmin
+  })
+}
+for (let i = 0; i < admins.length; i++) {
+  addUsers(i, admins, true)
+}
+for (let i = 0; i < names.length; i++) {
+  addUsers(i, names, false)
+}
+
+
+const orders = []
+const statuses = ['created', 'processing', 'cancelled', 'completed']
+//every user has a chance to have an order
+users.map(user => {
+  //max orders is 5
+  let numOrders = chance.integer({min: 0, max: 5})
+  let userId = user.id
+  for (let j = 0; j < numOrders ; j++) {
+    //If first order, create cart order, else create other order
+    let id = orders.length + 1
+    let status = j === 0 ? 'cart' : chance.pickone(statuses)
+    orders.push({
+      id,
+      status,
+      userId
+    })
+  }
+})
+
+const reviews = []
+//every watch has a review
+watches.map(watch => {
+  let watchId = watch.id
+  //every watch has at max 5 reviews
+  let reviewUsers = chance.pickset(users, chance.integer({min: 0, max: 5}))
+  reviewUsers.map(user => {
+    let userId = user.id
+    let rating = chance.integer({min: 1, max: 5})
+    let content = chance.paragraph()
+    reviews.push({
+      watchId,
+      userId,
+      rating,
+      content
+    })
+  })
+})
+
+const orderWatches = [];
+//every order has a watch
+orders.map(order => {
+  let orderId = order.id;
+  //every order has at max 5 watches
+  let randWatches = chance.pickset(watches, chance.integer({min: 0, max: 5}))
+  randWatches.forEach(watch => {
+    let watchId = watch.id
+    let fixedPrice = chance.integer({min: minPrice, max: maxPrice})
+    let quantity = chance.integer({min: 1, max: 4})
+    orderWatches.push({
+      watchId,
+      orderId,
+      fixedPrice,
+      quantity
+    })
+  })
+})
+
+const seed = () =>
+  Promise.all(watches.map(watch => Watch.create(watch)))
+  .then(() => Promise.all(users.map(user => User.create(user))))
+  .then(() => Promise.all(orders.map(order => Order.create(order))))
+  .then(() => Promise.all(reviews.map(review => Review.create(review))))
+  .then(() => Promise.all(orderWatches.map(orderWatch => OrderWatch.create(orderWatch))))
+
+const main = () => {
+  console.log('Syncing db...');
+  db.sync({
+      force: true
```

Comment:
This issues a DROP TABLE statement in SQL, right?

Reply:
Yes it clears the database for reseeding.